django/contrib/auth/views.py

```diff
@@ -185,7 +185,7 @@ def logout_then_login(request, login_url=None, extra_context=None):
     if not login_url:
         login_url = settings.LOGIN_URL
     login_url = resolve_url(login_url)
-    return logout(request, login_url, extra_context=extra_context)
+    return LogoutView.as_view(next_page=login_url, extra_context=extra_context)(request, login_url, extra_context)
```

Comment:
Okay, then lets not pass on the parameter to give the impression that it is used.